templates/broker.xml.erb

```diff
@@ -0,0 +1,96 @@
+        <bindings-directory>/var/lib/candlepin/activemq-artemis/bindings</bindings-directory>
+        <journal-directory>/var/lib/candlepin/activemq-artemis/journal</journal-directory>
+        <large-messages-directory>/var/lib/candlepin/activemq-artemis/largemsgs</large-messages-directory>
+        <paging-directory>/var/lib/candlepin/activemq-artemis/paging</paging-directory>
+
+        <!-- Use default thread pool max sizes as defined by Artemis. -->
+        <!-- Uncomment to customize.                                  -->
+        <!-- <thread-pool-max-sie>30</thread-pool-max-size> -->
+        <!-- <scheduled-thread-pool-max-size>5</scheduled-thread-pool-max-size> -->
+
+        <addresses>
+            <address name="event.default">
+                <multicast>
+                    <queue name="event.org.candlepin.audit.LoggingListener" />
+                    <queue name="event.org.candlepin.audit.ActivationListener" />
+                </multicast>
+            </address>
+
+            <address name="katello.candlepin">
+                <multicast>
+                  <queue name="katello_candlepin_event_monitor.candlepin_events" max-consumers="1" />
+                </multicast>
+            </address>
+        </addresses>
+
+        <address-settings>
+            <address-setting match="event.default">
+                <auto-create-queues>true</auto-create-queues>
+                <max-size-bytes>10485760</max-size-bytes>
+
+                <!-- By default, Artemis will page messages when the queue address is full. -->
+                <page-size-bytes>1048576</page-size-bytes>
+
+                <!--
+                    Configure message redelivery. We set the maximum number of times that a message should
+                    be redelivered to 0 so that messages will remain in the queue and will never get sent
+                    to the dead letter queue. Since candlepin does not currently set up, or use, a dead
+                    letter queue, any messages sent there will be lost. We need to prevent this.
+                -->
+                <redelivery-delay>30000</redelivery-delay>
+                <max-redelivery-delay>3600000</max-redelivery-delay>
+                <redelivery-delay-multiplier>2</redelivery-delay-multiplier>
+                <max-delivery-attempts>0</max-delivery-attempts>
+            </address-setting>
+
+            <address-setting match="katello.candlepin">
+                <auto-create-queues>false</auto-create-queues>
+                <max-size-bytes>10485760</max-size-bytes>
+
+                <!-- By default, Artemis will page messages when the queue address is full. -->
+                <page-size-bytes>1048576</page-size-bytes>
+
+                <redelivery-delay>0</redelivery-delay>
+                <max-delivery-attempts>1</max-delivery-attempts>
+            </address-setting>
+        </address-settings>
+
+        <diverts>
+            <divert name="katello_divert">
```

Comment:
I've updated to add the current level of filtering that we have today with qpid:

```
            <divert name="katello_divert">                                                                                                                    
                <exclusive>false</exclusive>                                                                                                                  
                <address>event.default</address>                                                                                                              
                <filter string="
                    (EVENT_TARGET='COMPLIANCE' and EVENT_TYPE='CREATED') OR                                                                       
                    (EVENT_TARGET='SYSTEM_PURPOSE_COMPLIANCE' and EVENT_TYPE='CREATED') OR                                                                    
                    (EVENT_TARGET='POOL' and EVENT_TYPE='CREATED') OR                                                                                         
                    (EVENT_TARGET='POOL' and EVENT_TYPE='DELETED') OR                                                                                         
                    (EVENT_TARGET='ENTITLEMENT' and EVENT_TYPE='CREATED') OR                                                                                  
                    (EVENT_TARGET='ENTITLEMENT' and EVENT_TYPE='DELETED')                                                                                     
                "/>                                                                                                                                           
                <forwarding-address>katello.candlepin</forwarding-address>                                                                                    
            </divert>
```